Resource/PYTHON/multilateration/analyze_data.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from scipy import signal
+
+
+# Functions for analyzing the data
+
+
+def calculate_xcorr_lag_array(data_array):
+    """Crosscorrelates the filtered data arrays and returns an array which
+    gives the number of samples between the measured signals at hyd1, hyd2, hyd3.
+
+    Args:
+        data_array: Array of filtered data from the hydrophones
+
+    Returns:
+        An array containing calculated cross-correlated lag between the hydrophones:
+        port_starboard_lag, port_stern_lag, starboard_stern_lag.
+    """
+
+    # Getting the values from data arrays
+    data_port = data_array[0]
+    data_starboard = data_array[1]
+    data_stern = data_array[2]
```

Comment:
This should be written in the docstring. The callee of this function has to ensure that `data_array[0]` actually is the data from the port hydrophone etc.
On the other hand it also doesn't really matter as it is just output as the lags from 0->1, 0->2, 1->2 at the end. But if this naming, e.g. `data_port` is used, it should also be mentioned at some point in the docstring.